sirepo/job_driver/sbatch.py

```diff
@@ -37,17 +37,12 @@ def __init__(self, req):
             # before it is overwritten by prepare_send
             _local_user_dir=pkio.py_path(req.content.userDir),
             _srdb_root=None,
+            # we allow one of each op type in. This is essentially a no-op (ha ha)
+            # but makes it easier to code the other cases.
+            cpu_slot_q=job_supervisor.SlotQueue(len(job_driver.OPS_WHICH_NEED_SLOTS)),
```

Comment:
Grammar nit - I'm not totally positive but I think "that" is better than "which" in this case since the phrase depends on need_slots (you use "that" for restrictive clauses).

Reply:
ha ha. The grammarian caught a bug: OPS_WHICH_NEED_SLOTS was not defined.

Reply:
374a8cb5